Mobile/MachineLearning/TextClassification/TextClassification/TextClassification/Models/TextClassification.cs

```diff
@@ -0,0 +1,181 @@
+            string[] words = str.ToLower().Split(' ');
+            len = words.Length;
+
+            for (cnt = 1; cnt <= len; cnt++)
+            {
+                /// Check words and fill the float array with proper values
+                if (ht.ContainsKey(words[cnt - 1]))
+                {
+                    value = Int32.Parse((string)ht[words[cnt - 1]]);
+                    float_array[cnt] = (float)value;
+                }
+                else
+                {
+                    Log.Debug(TAG, "Given word is not contained.");
+                    float_array[cnt] = (float)unknown;
+                }
+            }
+
+            /// Remainers are filled with 0
+            while (cnt < 256)
+            {
+                float_array[cnt++] = (float)pad;
+            }
+
+            return float_array;
+        }
+
+        /// <summary>
+        /// Load files and make hash table
+        /// </summary>
+        public void Init_model()
+        {
+            string label = ResourcePath + "models/labels.txt";
+            string vocab = ResourcePath + "models/vocab.txt";
+
+            if (File.Exists(label))
+            {
+                /// Load label file and tokenize
+                string rawLabel = File.ReadAllText(label);
+                tokenized_label = rawLabel.Split('\n');
+            }
+            else
+            {
+                Log.Error(TAG, "Label file is not exist");
+            }
+
+            if (File.Exists(vocab))
+            {
+                /// Load vocab file and tokenize
+                string rawVocab = File.ReadAllText(vocab);
+                tokenized_vocab = rawVocab.Split('\n');
+
+                int len = tokenized_vocab.Length;
+                /// Make vocab hash table to find word fast
+                for (int i = 0; i < len - 1; i++)
+                {
+                    string[] words = tokenized_vocab[i].Replace("\r", "").Split(' ');
+                    if (!ht.ContainsKey(words[0]))
+                    {
+                        ht.Add(words[0], words[1]);
+                    }
+                }
+            }
+            else
+            {
+                Log.Error(TAG, "Vocab file is not exist");
+            }
+        }
+
+        /// <summary>
+        /// Invoke text classification model and return result
+        /// </summary>
+        /// <param name="input"> float array which is converted from user input </param>
+        /// <returns> Probability of the paragraph being positive or negative </returns>
+        public float[] Invoke_mode(float[] input)
+        {
+            byte[] in_buffer = new byte[4 * 256];
+            byte[] out_buffer;
+            float[] output = new float[2];
+            TensorsInfo in_info;
+            TensorsInfo out_info;
+            TensorsData in_data;
+            TensorsData out_data;
+
+            string model_path = ResourcePath + "models/text_classification.tflite";
```

Comment:
```suggestion
            string model_path = Path.Combine(ResourcePath, "models/text_classification.tflite");
```
https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=netframework-4.8#System_IO_Path_Combine_System_String_System_String_

Reply:
Thank you. I will change it.